stix_shifter_modules/nozomi/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,555 @@
+{
+  "ip_src": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "ipv6-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "network-traffic.src_ref",
+      "object": "nt",
+      "references": "src_ip"
+    },
+    {
+      "key": "x-ibm-finding.src_ip_ref",
+      "object": "alert",
+      "references": "src_ip"
+    }
+  ],
+  "ip_dst": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "ipv6-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "network-traffic.dst_ref",
+      "object": "nt",
+      "references": "dst_ip"
+    },
+    {
+      "key": "x-ibm-finding.dst_ip_ref",
+      "object": "alert",
+      "references": "dst_ip"
+    }
+  ],
+  "mac_src": [
+    {
+      "key": "mac-addr.value",
+      "object": "mac_addr_src"
+    },
+    {
+      "key": "network-traffic.src_ref",
+      "object": "nt",
+      "references": "mac_addr_src"
```

Comment:
so it replaces the src_ref that is created by ip_src. you may need to create separate network traffic object if there are two sources such as ip and mac

Reply:
ip and mac addresses are from same source. We have observed that sometimes in alert only one of these attributes is filled with the value otherwise it is null. Hence we have added the references in both cases.